### PR TITLE
remove global namespaces

### PR DESCRIPTION
**Pull request by floweisshardt**
this allows the people detection to be started multiple times, e.g. once for each camera in a left and right camera setup.

@ipa-rmb: please review

### CODE REVIEW

File: cob_people_detection/ros/launch/detection_tracker.launch [hunk 1]
```diff
@@ -5,12 +5,9 @@
   <!-- <node pkg="cob_people_detection" ns="cob_people_detection/people_segmentation" type="openni_tracker" name="openniTrackerNode" output="screen"/> -->
 
   <!-- detection tracker node (tracks faces in color and depth image and publishes their positions) -->
-  <rosparam command="load" ns="/cob_people_detection/detection_tracker" file="$(find cob_people_detection)/ros/launch/detection_tracker_params.yaml"/>
-  <node name="detection_tracker" pkg="cob_people_detection" ns="/cob_people_detection/detection_tracker" type="detection_tracker_node" output="screen">
-    <remap from="face_position_array_in" to="/cob_people_detection/face_recognizer/face_recognitions"/>
-    <remap from="people_segmentation_image" to="/cob_people_detection/people_segmentation/people_segmentation_image"/>
```
Comment (ipa-rmb) [inline cob_people_detection/ros/launch/detection_tracker.launch:11]: Why did you remove this remapping? I think this belongs to the link to the Asus skeleton tracking software and can incorporate detected 3d humans. If there are no good reasons why it was removed, please leave it in commented.

File: cob_people_detection/ros/launch/face_capture.launch [hunk 2]
```diff
@@ -2,13 +2,10 @@
 
 <launch>
   <!-- face capture node (captures faces and stores them in the database) -->
-  <rosparam command="load" ns="/cob_people_detection/face_capture" file="$(find cob_people_detection)/ros/launch/face_recognizer_params.yaml"/>
-  <node name="face_capture" pkg="cob_people_detection" ns="/cob_people_detection/face_capture" type="face_capture_node" output="screen"><!-- launch-prefix="gdb -ex run args"--><!--launch-prefix="valgrind"-->
-    <remap from="face_detections" to="/cob_people_detection/face_detector/face_positions"/>
-    <!--remap from="color_image" to="/cob_people_detection/image_flip/colorimage_out"/--> <!-- only activate on cob3 robots -->
-    <remap from="color_image" to="/cob_people_detection/sensor_message_gateway/colorimage_out"/>
```
Comment (ipa-rmb) [inline cob_people_detection/ros/launch/face_capture.launch:9]: Has the remapping of the incoming color image been removed? Please leave it there with the right namespace otherwise it is impossible to train new people. Or did you solve this in another way?

File: cob_people_detection/ros/launch/head_detector.launch [hunk 3]
```diff
@@ -1,15 +1,11 @@
 <?xml version="1.0"?>
 
 <launch>
-  <!-- head detection node (detects faces in depth image and publishes their positions) -->
-  <rosparam command="load" ns="/cob_people_detection/head_detector" file="$(find cob_people_detection)/ros/launch/head_detector_params.yaml"/>
-  <node name="head_detector" pkg="cob_people_detection" ns="/cob_people_detection/head_detector" type="head_detector_node" output="screen">
-    <!--remap from="pointcloud_rgb" to="/cam3d/depth/upright/points"/-->
-    <!--remap from="pointcloud_rgb" to="/cob_people_detection/image_flip/pointcloud_rgb_out"/-->  <!-- only activate on cob3 robots -->
```
Comment (ipa-rmb) [inline cob_people_detection/ros/launch/head_detector.launch]: Please do not delete such valuable comments. If we ever want to use this again on a cob3 then it would take significantly more time if I have to figure out how image flip was integrated into the pipeline.

File: cob_people_detection/ros/launch/face_capture.launch [hunk 4]
```diff
@@ -2,13 +2,10 @@
 
 <launch>
   <!-- face capture node (captures faces and stores them in the database) -->
-  <rosparam command="load" ns="/cob_people_detection/face_capture" file="$(find cob_people_detection)/ros/launch/face_recognizer_params.yaml"/>
-  <node name="face_capture" pkg="cob_people_detection" ns="/cob_people_detection/face_capture" type="face_capture_node" output="screen"><!-- launch-prefix="gdb -ex run args"--><!--launch-prefix="valgrind"-->
-    <remap from="face_detections" to="/cob_people_detection/face_detector/face_positions"/>
-    <!--remap from="color_image" to="/cob_people_detection/image_flip/colorimage_out"/--> <!-- only activate on cob3 robots -->
```
Comment (ipa-rmb) [inline cob_people_detection/ros/launch/face_capture.launch]: Please keep those kinds of comments as they will help me whenever I need to run this on a cob3 again.

File: cob_people_detection/ros/launch/people_detection.launch [hunk 5]
```diff
@@ -20,28 +19,23 @@ for Care-O-bot default is cam3d_nodelet_manager -->
       <arg name="colorimage_in_topic" value="$(arg colorimage_in_topic)"/>
       <arg name="pointcloud_rgb_in_topic" value="$(arg pointcloud_rgb_in_topic)"/>
     </include>
-    <!--include file="$(find cob_people_detection)/ros/launch/image_flip.launch"/--> <!-- only activate on cob3 robots -->
```
Comment (ipa-rmb) [inline cob_people_detection/ros/launch/people_detection.launch]: keep

File: cob_people_detection/ros/launch/people_detection.launch [hunk 6]
```diff
@@ -20,28 +19,23 @@ for Care-O-bot default is cam3d_nodelet_manager -->
...
   <group if="$(arg using_nodelets)">
     <!-- special launch files for usage with nodelets -->
     <node if="$(arg start_manager)" pkg="nodelet" type="nodelet" name="$(arg nodelet_manager)"  args="manager" output="screen"/>
     <include file="$(find cob_people_detection)/ros/launch/sensor_message_gateway_nodelet.launch">
+      <arg name="nodelet_namespace" value="$(arg camera_namespace)"/>
       <arg name="nodelet_manager" value="$(arg nodelet_manager)"/>
       <arg name="colorimage_in_topic" value="$(arg colorimage_in_topic)"/>
       <arg name="pointcloud_rgb_in_topic" value="$(arg pointcloud_rgb_in_topic)"/>
     </include>
-    <!-- include file="$(find cob_people_detection)/ros/launch/image_flip_nodelet.launch">
```
Comment (ipa-rmb) [inline cob_people_detection/ros/launch/people_detection.launch]: keep

File: cob_people_detection/ros/launch/people_detection.launch [hunk 7]
```diff
@@ -20,28 +19,23 @@ for Care-O-bot default is cam3d_nodelet_manager -->
...
-    	<arg name="nodelet_manager" value="$(arg nodelet_manager)"/>
-    </include--> <!-- only activate on cob3 robots -->
   </group>
```
Comment (ipa-rmb) [inline cob_people_detection/ros/launch/people_detection.launch]: keep

File: cob_people_detection/ros/launch/people_detection.launch [hunk 8]
```diff
@@ -20,28 +19,23 @@ for Care-O-bot default is cam3d_nodelet_manager -->
...
     <!-- special launch files for usage with nodelets -->
     <node if="$(arg start_manager)" pkg="nodelet" type="nodelet" name="$(arg nodelet_manager)"  args="manager" output="screen"/>
     <include file="$(find cob_people_detection)/ros/launch/sensor_message_gateway_nodelet.launch">
+      <arg name="nodelet_namespace" value="$(arg camera_namespace)"/>
       <arg name="nodelet_manager" value="$(arg nodelet_manager)"/>
       <arg name="colorimage_in_topic" value="$(arg colorimage_in_topic)"/>
       <arg name="pointcloud_rgb_in_topic" value="$(arg pointcloud_rgb_in_topic)"/>
     </include>
-    <!-- include file="$(find cob_people_detection)/ros/launch/image_flip_nodelet.launch">
-    	<arg name="nodelet_manager" value="$(arg nodelet_manager)"/>
```
Comment (ipa-rmb) [inline cob_people_detection/ros/launch/people_detection.launch]: keep

File: cob_people_detection/ros/launch/people_detection_display.launch [hunk 9]
```diff
@@ -4,18 +4,16 @@
   <arg name="display_results_with_image_view" default="true"/>   <!-- set to false if you do not like to display the camera image with names attached to detected faces within an image_view window -->
 
   <!-- display node for the detections -->
-  <rosparam command="load" ns="cob_people_detection/people_detection_display" file="$(find cob_people_detection)/ros/launch/people_detection_display_params.yaml"/>
-  <node pkg="cob_people_detection" ns="cob_people_detection/people_detection_display" type="people_detection_display_node" name="people_detection_display" output="screen">
-    <remap from="face_position_array" to="/cob_people_detection/detection_tracker/face_position_array"/>
-    <remap from="face_detections" to="/cob_people_detection/face_detector/face_positions"/>
-    <!--remap from="pointcloud_in" to="/cob_people_detection/image_flip/pointcloud_rgb_out"/-->
-    <!--remap from="colorimage_in" to="/cob_people_detection/image_flip/colorimage_out"/--> <!-- only activate on cob3 robots -->
```
Comment (ipa-rmb) [inline cob_people_detection/ros/launch/people_detection_display.launch]: keep

**Comment by ipa-rmb** [comment]
Check the comments, rest is fine.

File: cob_people_detection/ros/launch/detection_tracker.launch (hunk 1 above)
Comment (floweisshardt) [inline cob_people_detection/ros/launch/detection_tracker.launch:11]: the `cob_people_detection` namespace does no longer exist. All topics are now defined relative to the root ns but without global `/`. so no remapping needed here

File: cob_people_detection/ros/launch/face_capture.launch (hunk 2 above)
Comment (floweisshardt) [inline cob_people_detection/ros/launch/face_capture.launch:9]: well spottet. I added it again.

File: cob_people_detection/ros/launch/detection_tracker.launch (hunk 1 above)
Comment (ipa-rmb) [inline cob_people_detection/ros/launch/detection_tracker.launch:11]: Is this input or output. I assumed it would be an input topic from a people segmentation software. Then please leave it in as a comment. I know the namespace would be wrong now, but it at least tells that there is some topic of this kind received without reading all source code.

Comment (floweisshardt) [inline cob_people_detection/ros/launch/detection_tracker.launch:11]: readded as a comment
